hoophq/hoop
# [FEAT] Jira Integration

Implementations:

- Jira_integration table in database;
- Jira endpoints POST, GET and PUT
- Jira integrations with steps of one session;
  - When creating a session; Create a Jira issue
  - When updating a session with review; Update the Jira issue
  - When the session is ready to execute; Update the Jira issue
  - When the session was executed; Update the Jira issue

**How to test**
Everyone was invited to the JIRA board that I created. (If you need a new invite, let me know)

In the POST endpoint `/jira-integration` you send this body
```
{
  "jira_url": "https://usehoop.atlassian.net",
  "jira_user": "rogerio@usehoop.com",
  "jira_api_token": "", // Ask me for the key
  "jira_project_key": "TESTE",
  "status": "enabled"
}
```

Now you only need to execute a session and see the board TESTE to see the card being created.
